content/decadal/1_forecasts.md

```diff
@@ -0,0 +1,22 @@
+---
+tag: Forecasts
+title: Decadal climate predictions in EUCP
+img: lead_center.png
+caption: The Lead Centre for Annual-to-Decadal Climate Prediction
+datasets:
+  - title: TBA
+    contact:
+      - name:
+        orcid:
+    workpackage:
+    licencse:
+    format:
+    size:
+    doi: N/A
+    details: N/A
+---
+The EUCP project has been pivotal in establishing the [WMO Lead Centre for annual to decadal climate prediction](https://hadleyserver.metoffice.gov.uk/wmolc/), where all the latest information about existing global climate prediction systems is collected and evaluated to identify the main factors that could improve their societal impact. These climate predictions from leading organisations around the world help us identify the relative merits of decadal predictions from different sources, and provide guidance on the use of existing climate predictions.
+
+Decadal climate predictions forecast the future evolution of climate for timescales ranging from a month to a decade into the future. The main characteristic of these predictions is that they require accurate information about the present (or ‘initial’) state of the climate system since it provides some predictability for the future climate. They are therefore often termed ‘initialised’ simulations. Decadal climate predictions come with their challenges: they require sets of hindcasts that are very expensive to run (these test the model by running it over a period in the past and comparing the simulations to observations), strategies to replicate observational uncertainty, and they strongly rely on a consistent observation system. However, improving decadal climate forecasting is crucial, since it covers vital timescales for decision makers, such as climate predictions for the next few years ahead.
+
+Please visit the website of the [The Lead Centre for Annual-to-Decadal Climate Prediction](https://hadleyserver.metoffice.gov.uk/wmolc/) for an up to date overview of the latest climate prediction data.
```

Comment:
In this case, the "list of datasets" header below doesn't really make sense. We could remove it altogether, or put this last sentence under a renamed "Data access" header.

Reply:
Good point! Let's use "Data access".